tools/c7n_policystream/policystream.py

```diff
@@ -399,11 +399,7 @@ def _process_stream_commit(self, change):
                     current_policies += self.policy_files[f]
             elif delta.status == GIT_DELTA_INVERT['GIT_DELTA_DELETED']:
                 if f in self.policy_files:
-                    # if the policies were moved, only add in policies
```

Comment:
I think the intent was for policies moved in the same commit, where as the tests added here model remove and add in separate commits

Reply:
The second test moves a policy from the removed file to another file.

What do you mean by "moved in the same commit" if it isn't the second test case?

Reply:
there's a couple cases for move, a simple file rename, or a content change move. i added an additional test case for the rename, but the implementation change looks good.